blogs/2016/04/22/edge-debugging-for-vs-code.md

```diff
@@ -0,0 +1,48 @@
+---
+Order: 2
+TOCTitle: Microsoft Edge Debugging for VS Code
+PageTitle: Microsoft Edge Debugging for VS Code
+MetaDescription: Microsoft Edge Debugging for Visual Studio Code
+Date: 2016-04-22
+ShortDescription: As a part of our continuous effort to simplify the daily workflow for developers, we are happy to enable web developers to debug their code running inside Microsoft Edge, directly from the editor by introducing Microsoft Edge Debugging for Visual Studio Code.
+
+Author: Kenneth Auchenberg
+---
+
+# Introducing Chrome Debugging for VS Code
```

Comment:
Copy and paste error "Introducing Edge Debugging..."
I would also remove the "for VS Code" as it makes the h1 header too long.